### ISSUE TITLE
NTR: Abnormal/Increased/Decreased proportion of immature gamma-delta T cell

### ISSUE THREAD

**Issue by LCCarmody** [obophenotype/human-phenotype-ontology#5778]
**Preferred term label:**

Abnormal proportion of immature gamma-delta T cell | CL:0000799 | immature gamma-delta
-- | -- | --

Adding these terms in response to proportional cell discussion.

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by LCCarmody** [comment]
added terms #5779 PR
